AsyncDisplayKit/ASDisplayNode.mm

```diff
@@ -3857,6 +3857,9 @@ - (void)addSubnode:(ASDisplayNode *)subnode
     if (selfNode) {
       [selfNode addSubnode:subnode];
     } else {
+      if (subnode.supernode) {
```

Comment:
@rcancro thanks for submitting this!  I'm curious how you found it.

Could you also fix the CALayer -addSubnode: category method that is just beneath this?